content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md

```diff
@@ -100,10 +100,19 @@ When creating a group, you must choose a policy that defines which organizations
 You can update the access policy of a runner group, or rename a runner group.
 
 {% data reusables.github-actions.self-hosted-runner-configure-runner-group-access %}
+    
+## Creating a self-hosted runner in a group
+You can use the configuration script on the self-hosted runner to create it in a runner group. For example, this command creates a self-hosted runner in the runner group `rg-runnergroup`.
+
+```sh
+./config.sh --url $org --token $token --runnergroup rg-runnergroup
```

Comment:
--url $org? Shouldn't that be the runner registration url

Reply:
You're right, the way I've written it is not precise enough. It needs to be the url of the enterprise or org in which you're registering the runner.

How about being very specific:

```sh
./config.sh --url $org_or_enterprise_url --token $token --runnergroup rg-runnergroup
```